src/test/scala/tech/sourced/gemini/CassandraSparkSpec.scala

```diff
@@ -54,7 +54,7 @@ class CassandraSparkSpec extends FlatSpec
     val gemini = Gemini(sparkSession)
 
     println("Hash")
-    gemini.hashAndSave("src/test/resources/siva")
+    gemini.hashAndSave("src/test/resources/siva", 0)
```

Comment:
A [default parameter value](https://docs.scala-lang.org/tour/default-parameter-values.html) can be added to a function definition, to avoid extra API changes.